subworkflows/nf-neuro/preproc_t1/main.nf

```diff
@@ -51,9 +65,10 @@ workflow PREPROC_T1 {
         }
 
         else {
-            ch_template = ch_template.ifEmpty(Channel.error('Template is required for ANTS registration'))
-            ch_probability_map = ch_probability_map.ifEmpty(Channel.error('Probability map is required for ANTS registration'))
-            ch_bet = IMAGE_RESAMPLE.out.image.join(ch_template).join(ch_probability_map)
+            ch_bet = IMAGE_RESAMPLE.out.image
+                .join(ch_template)
+                .join(ch_probability_map)
```

Comment:
Why did you remove the `remainder: true` ?

Reply:
Don't know if this was answered. I assume yes, but will give a reason for posterity : the template and probabilities aren't optional for BETCROP_ANTSBET, so we should not have to `remainder` here.

I would agree to add the error checking. However, I also thinks it is not the job of the subworkflow to do input validation. It's the job of the main pipeline.

Reply:
Agreed! I was more curious about the sudden removal of the `remainder`, but that makes perfect sense!

Reply:
I remember we had the conversation of removing the ifempty checks. Would you like me to add them back?

Reply:
No, keep that like that for now